### DISCUSSION TITLE
APRS Decoder advice

### DISCUSSION THREAD

**Discussion by W7DGP** [alanbjohnston/CubeSatSim#262] [General]
So far, my CubeSatSim is running very well. What I would like to do is to test the APRS/AFSK mode. I am not all that familiar with APRS but I understand that valid APRS data is transmitted. All I need is a way to decode it. Having Googled around, all I find are apps to make a complete APRS station with digipeater, etc. I'm using a Win 11 laptop as my ground station without SDR # or HDSDR - just FoxTelem. 

I'm looking for some recommendations.

Thanks,
Larry W7DGP

**Comment by alanbjohnston** [comment]
Hi Larry, 

Here’s some instructions on how to setup the AFSK/APRS ground station on Windows:

https://github.com/alanbjohnston/CubeSatSim/wiki/Decoding-APRS-AFSK-Telemetry

You could also use SoundModem or Direwolf instead of AFSK 1200 Decoder, although each of them require a little more configuration.

Good luck!

73,
Alan 
KU2Y

**Comment by W7DGP** [comment]
Thank you Alan!
I did it the simple way using the command line and multimon-ng.  I let it run for a while and copy/pasted into your spreadsheet. Awesome! This is the first time since building the cubesat last year that I was able to decode the APRS data. I had no idea it was sat housekeeping data. However, I see in each line there is geolocation info like lat-lon. Where did that come from ..... or did I enter it when I entered my callsign? Anyway, each frame looks like this:  APRS: W7DGP-11>APCSS:=4745.36N\12212.19WShi hi 100 199 199 199 299 299 299 268 383 350 300 300 446 400 400 400 500 500 500 500 600 600 600 650 OK BME280 21.46 1011.26 16.58 30.60 M U6050 -0.39 -0.09 0.05 0.13 0.02 0.94 XS 16.17 0.00.  I'm assuming the hihi is the start of data similar to CW data on the sats in orbit.
So, I'm going to learn some more about APRS. My only experience was with a guy at a hamfest having it in his handheld. I went to aprs.fi on my phone and saw his location on the map. That all I know....for now. 
Thanks again,
Larry W7DGP
